low-level/files/opt/seagate/sspl/setup/cortx_manifest/lr2/manifest_storage.py

```diff
@@ -0,0 +1,381 @@
+#!/bin/python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com
+
+"""
+ ***************************************************************************
+  Description: Storage class provides manifest information like technical
+  information of storage/enclosure components etc,.
+ ***************************************************************************
+"""
+
+import errno
+import json
+import re
+import time
+
+from framework.utils.conf_utils import GLOBAL_CONF, Conf, STORAGE_TYPE_KEY
+from manifest_error import ManifestError
+from cortx_manifest import CortxManifest
+from framework.base.sspl_constants import MANIFEST_SVC_NAME
+from framework.utils.service_logging import CustomLog, logger
+from framework.platforms.realstor.realstor_enclosure import (
+    singleton_realstorencl as ENCL)
+
+
+class ManifestStorage(CortxManifest):
+    """Provides storage manifest related information."""
+
+    name = "manifest_storage"
```

Comment:
Disocvery lib(resource.py) is importing class based on node name in the rpath. So this must be storage as per rpath. Please align to that.

Reply:
done